pkgs/top-level/python-packages.nix

```diff
@@ -10510,15 +10510,24 @@ in modules // {
 
   flake8 = buildPythonPackage rec {
     name = "flake8-${version}";
-    version = "2.5.4";
+    version = "3.0.4";
 
     src = pkgs.fetchurl {
       url = "mirror://pypi/f/flake8/${name}.tar.gz";
-      sha256 = "0bs9cz4fr99r2rwig1b8jwaadl1nan7kgpdzqwj0bwbckwbmh7nc";
+      sha256 = "03cpdrjxh0fyi2qpdxbbrmxw7whiq3xr3p958gr6yzghk34i1hml";
     };
 
     buildInputs = with self; [ nose mock ];
```

Comment:
according to this [link](http://nixos.org/nix/manual/#sec-conf-file) it does work on OSX.